package.json

```diff
@@ -8,7 +8,11 @@
     "type": "git",
     "url": "git+https://github.com/across-protocol/across-smart-contracts-v2.git"
   },
-  "dependencies": {},
+  "dependencies": {
+    "@openzeppelin/contracts": "^4.4.2",
```

Comment:
why shouldn't these be dev dependencies?

Reply:
I made them normal dependencies because that's what we do in the V1 repo: https://github.com/across-protocol/across-smart-contracts/blob/master/package.json